src/meshes/t8code_mesh.jl

```diff
@@ -1263,6 +1428,64 @@ function fill_mesh_info!(mesh::T8codeMesh, interfaces, mortars, boundaries,
     return nothing
 end
 
+function get_levels(mesh::T8codeMesh)
+    return trixi_t8_get_local_element_levels(mesh.forest)
+end
+
+function get_cmesh_info(mesh::T8codeMesh)
+    @assert t8_forest_is_committed(mesh.forest) != 0
+    cmesh = t8_forest_get_cmesh(mesh.forest)
+    return get_cmesh_info(cmesh, ndims(mesh))
+end
+
+function get_cmesh_info(cmesh::Ptr{t8_cmesh}, ndims)
+    num_trees = t8_cmesh_get_num_trees(cmesh)
```

Comment:
This and all other data are global?

Reply:
Yes! Every MPI ranks has a full copy of `cmesh`. Added a comment explaining that.